<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>busca_externo</class>
 <widget class="QDialog" name="busca_externo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buscar Externo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Búsqueda:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Criterio:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="criterio_comboBox">
       <item>
        <property name="text">
         <string>Comience</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Contenga</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Campo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="campo_comboBox">
       <item>
        <property name="text">
         <string>Nombre/Razón social</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nombre comercial</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Código</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Domicilio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Población</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Provincia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>País</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>email</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refrescar_pushButton">
       <property name="text">
        <string>&amp;Refrescar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/rayo.png</normaloff>:/iconos/graf/rayo.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Código</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre/Razón</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre Comercial</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Domicilio</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Población</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Provincia</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CP</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>País</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelar_pushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptar_pushButton">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/check2.png</normaloff>:/iconos/graf/check2.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>criterio_comboBox</tabstop>
  <tabstop>campo_comboBox</tabstop>
  <tabstop>refrescar_pushButton</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>cancelar_pushButton</tabstop>
  <tabstop>aceptar_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelar_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>busca_externo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aceptar_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>busca_externo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
